cmd/justgo/justgo.go

```diff
@@ -51,7 +62,20 @@ func buildProject(path string) {
 
 	fileUrl := "https://github.com/inadarei/justgo/archive/master.zip"
 	tmpFilePath := os.TempDir() + "justgo.zip"
-	//fmt.Println("Downloading to ", tmpFilePath)
+
+	// Move all conflicting files to tmp dir and move them back post-build
+	filesToMove := getConflictingFiles(path)
+	if filesToMove != nil {
+		for _, file := range filesToMove {
+			srcPath := filepath.Join(path, file)
+			tmpFilePath := filepath.Join(os.TempDir(), file)
```

Comment:
By naming this variable tmpFilePath (which was already used for a different purpose on line 64) you are causing confusion and possibility of conflict.